experiment_scheduler/master/master.py

```diff
@@ -31,6 +32,10 @@
 )
 from experiment_scheduler.master.process_monitor import ProcessMonitor
 
+from experiment_scheduler.db_util.connection import initialize_db
+from experiment_scheduler.db_util.task_manager import TaskManager
```

Comment:
It seem a task manager service class not DB adapter. Could you import it with `as`?
For example, 
`from experiment_scheduler.db_util.task_manager import TaskManager as TaskManagerDB`